lib/tasks/normalize_resources.rake

```diff
@@ -0,0 +1,12 @@
+namespace :data do
+  desc 'Update resources with nil quantity/used fields'
+  task normalize_resources: :environment do
+    Resource.where('used is null or quantity is null').each do |resource|
```

Comment:
Please fix this where statement according to the guide:
http://guides.rubyonrails.org/active_record_querying.html#array-conditions